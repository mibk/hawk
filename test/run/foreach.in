apple	11
orange	12
pear	2
apple	3
apple	8
pear	3
