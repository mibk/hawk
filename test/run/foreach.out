apple 22
orange 12
pear 5
